### PR TITLE
Special logo with Javascript

### PR DESCRIPTION
**Pull request by pllim**
Follow up of #316 

Close #317 

It works when I load the HTML locally. Feel free to test it yourselves before merge.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -16,6 +16,29 @@
 
 <!-- Google analytics -->
 <script src="js/analytics.js"></script>
+
+<!-- Meow woof -->
+<script>
+window.onload = function() {
+  var today = new Date();
+  var month = today.getMonth() + 1;
```
Comment (pllim) [inline index.html]: Apparently JS zero-indexed their month! :scream: 

https://stackoverflow.com/questions/23161366/javascript-get-current-day-and-month-issue

**Comment by bsipocz** [comment]
needs a rebase

**Comment by pllim** [comment]
I'll also have to add those files deleted in #317 back in. Are you sure you want to go on with this?

**Comment by bsipocz** [comment]
it was good fun. But agreed, maybe we need @eteq to merge this.

**Pull request merged**